test/studies/shootout/reverse-complement/bharshbarg/revcomp-begin.chpl

```diff
@@ -1,73 +1,69 @@
+param newLineChar = 0x0A;
+param greaterThan = 0x3E;
 const pairs = "ATCGGCTAUAMKRYWWSSYRKMVBHDDHBVNN\n\n";
 var table : [1..128] uint(8);
 
-proc main(args: [] string) {
-  param newLineChar = 0x0A;
-  param greaterThan = 0x3E;
-  var inFile = openfd(0); // stdin
-  const fileLen = inFile.length();
-  var data : [1..fileLen] uint(8);
-  var r = inFile.reader(locking=false);
-
+proc main() {
   // initialize complement table
   for i in 0..#pairs.length by 2 {
-    table[pairs.buff[i]] = pairs.buff[i+1];      // uppercase
+    table[pairs.buff[i]] = pairs.buff[i+1];        // uppercase
     if pairs.buff[i] != newLineChar then
```

Comment:
Looking at it again, we'd have to do the same awkward index conversion here too.